### PR TITLE
Folder structure changes

### PR DESCRIPTION
**Pull request by jennifererwangg**
# Folder Structure Changes

## Description
Reorganized the folder structure for all models in the zoo. This enables easier navigation for users.

### CODE REVIEW

**Comment by prasanthpul** [comment]
Thanks for this PR. In the current model zoo, models are in multiple places (subfolders under "models" folder and folders at root level). This organization provides a common structure for all models, organized around input type and task.

File: README.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ Face detection models identify and/or recognize human faces in images. Some more
 
 |Model Class |Reference |Description |
 |-|-|-|
```
Comment (prasanthpul) [inline README.md]: consider shortening to "body_analysis" instead of "Body_Face_and_Gesture_Analysis"

File: README.md [hunk 2]
```diff
@@ -72,9 +72,9 @@ Face detection models identify and/or recognize human faces in images. Some more
...
-|<b>[ArcFace](models/face_recognition/ArcFace/)</b>|[Deng et al.](https://arxiv.org/abs/1801.07698)|ArcFace is a CNN based model for face recognition which learns discriminative features of faces and produces embeddings for input face images.|
+|<b>[ArcFace](Image/Body_Face_and_Gesture_Analysis/ArcFace)</b>|[Deng et al.](https://arxiv.org/abs/1801.07698)|ArcFace is a CNN based model for face recognition which learns discriminative features of faces and produces embeddings for input face images.|
```
Comment (prasanthpul) [inline README.md]: consider shortening to "body_analysis" instead of "Body_Face_and_Gesture_Analysis"

File: README.md [hunk 3]
```diff
@@ -29,41 +29,41 @@ This collection of models take images as input, then classifies the major object
...
 ### Object Detection & Image Segmentation <a name="object_detection"/>
 Object detection models detect the presence of multiple objects in an image and segment out areas of the image where the objects are detected. Semantic segmentation models partition an input image by labeling each pixel into a set of pre-defined categories.
 
 |Model Class |Reference |Description |
 |-|-|-|
-|<b>[Tiny_YOLOv2](tiny_yolov2)</b>|[Redmon et al.](https://arxiv.org/pdf/1612.08242.pdf)|Deep CNN model for Object Detection|
-|<b>[SSD](ssd)</b>|[Liu et al.](https://arxiv.org/abs/1512.02325)|Deep CNN model for Object Detection|
-|<b>[Faster-RCNN](faster_rcnn)</b>|[Ren et al.](https://arxiv.org/abs/1506.01497)|Deep CNN model for Object Detection|
-|<b>[Mask-RCNN](mask_rcnn)</b>|[He et al.](https://arxiv.org/abs/1703.06870)|Deep CNN model for Object Segmentation|
+|<b>[Tiny YOLOv2](Image/object_detection_and_image_segmentation/Tiny_YOLOv2)</b>|[Redmon et al.](https://arxiv.org/pdf/1612.08242.pdf)|Deep CNN model for Object Detection|
```
Comment (prasanthpul) [inline README.md]: consider shortening to "object_detection_segmentation" instead of "object_detection_and_image_segmentation"

File: README.md [hunk 4]
```diff
@@ -29,41 +29,41 @@ This collection of models take images as input, then classifies the major object
...
+|<b>[ZFNet-512](image/classification/zfnet-512)</b>|[Zeiler et al.](https://arxiv.org/abs/1311.2901)|Deep CNN model (up to 8 layers) for Image Classification that tuned the hyperparameters of AlexNet and won the ImageNet Challenge in 2013. <br> Top-5 error from paper - ~14.3%|
 <hr>
 
 #### Domain-based Image Classification <a name="domain_based_image"/>
 This subset of models classify images for specific domains and datasets.
 
 |Model Class |Reference |Description |
 |-|-|-|
-|[**MNIST**- Handwritten Digit Recognition](mnist) |[Convolutional Neural Network with MNIST](https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_103D_MNIST_ConvolutionalNeuralNetwork.ipynb)	|Deep CNN model for handwritten digit identification|
+|<b>[MNIST-Handwritten Digit Recognition](image/classification/mnist-handwritten_digit_recognition)</b>|[Convolutional Neural Network with MNIST](https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_103D_MNIST_ConvolutionalNeuralNetwork.ipynb)	|Deep CNN model for handwritten digit identification|
```
Comment (prasanthpul) [inline README.md]: subfolder should just be "mnist"

File: vision/classification/extact_imagenet.py [hunk 5]
```diff
@@ -81,4 +81,4 @@ def main():
     extract_val(val_tar_fname, os.path.join(target_dir, 'val'))
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (prasanthpul) [inline vision/classification/extact_imagenet.py]: looks like the line ending got changed

Comment (jennifererwangg) [reply vision/classification/extact_imagenet.py]: I tried overwriting it with the original file but it shows that there is no difference between the original and my revision.

**Comment by ebarsoum** [comment]
@abhinavs95 why switch from image to vision? So now, image and video models will be in the same folder? Would that mean we need to separate audio and speech?

**Comment by abhinavs95** [comment]
@ebarsoum I am fine with either name, I suggested vision since its common nomenclature used in most projects. If we go forward with vision we would need to put both image and video models in it.

@jennifererwangg thanks for making the changes, LGTM.

**Comment by ebarsoum** [comment]
@jennifererwangg can you fix the conflict? Once you do, I will merge the change.

**Pull request merged**